src/main/resources/templates/laDigitalAssister/householdIncomeWho.html

```diff
@@ -22,7 +22,7 @@
                             content=~{::householdMembers})}">
                 <th:block th:ref="householdMembers">
                   <th:block
-                      th:replace="~{fragments/inputs/radio :: radio(inputName=${inputName},value='you', label=${inputData.firstName + ' ' + inputData.lastName + ' (you) '})}"/>
+                      th:replace="~{fragments/inputs/radio :: radio(inputName=${inputName},value='you', label=${inputData.firstName + ' ' + inputData.lastName + ' ' + #messages.msg('general.you')})}"/>
```

Comment:
This was causing issues with the spanish translation since the text always appeared as "(you)"